docs/integrations/duckdb.md

```diff
@@ -9,14 +9,13 @@ has_children: false
 
 # Using lakeFS with DuckDB
 
-[DuckDB](https://duckdb.org/){:target="_blank"} is an in-process  SQL OLAP database management system.
+[DuckDB](https://duckdb.org/){:target="_blank"} is an in-process SQL OLAP database management system.
+
 {% include toc.html %}
 
 ## Configuration
 
-Querying data in lakeFS from DuckDB is similar to querying data in S3 from DuckDB. It is done using the [httpfs extension](https://duckdb.org/docs/extensions/httpfs.html){:target="_blank"}.
-
-### Configure the lakeFS S3 Gateway endpoint
+Querying data in lakeFS from DuckDB is similar to querying data in S3 from DuckDB. It is done using the [httpfs extension](https://duckdb.org/docs/extensions/httpfs.html){:target="_blank"} connecting to the S3 Gateway that lakeFS provides.
```

Comment:
Perhaps we should link to an explanation on the s3 gateway. Could be in a different PR.

Reply:
Done